### DISCUSSION TITLE
Port forward photoprism

### DISCUSSION THREAD

**Discussion by MrJoki007** [photoprism/photoprism#2548] [Q&A]
Nevermind, it was not user authentification it was https

**Comment by graciousgrey** [comment]
If you are not running PhotoPrism in public mode, there was an admin user to authenticate from the beginning. 

> Is it now safe, to open the port of photoprism to the internet to access it outside of my home network?

If you install PhotoPrism on a public server outside your home network, ALWAYS run it behind a secure HTTPS reverse proxy. Your files and passwords will otherwise be transmitted in clear text and can be intercepted by anyone, including your provider, hackers, and governments. Backup tools and file sync apps may refuse to connect as well.

**Reply by MrJoki007** [reply]
Is there a simple way for https encryption on TrueNAS core, because ive never worked with a reverse proxy

**Reply by graciousgrey** [reply]
Maybe a community member can help with TrueNAS specific questions.
In general we recommend using Traefik: https://docs.photoprism.app/getting-started/proxies/traefik/
